### PR TITLE
feature/AK-36: Added AWS EFS CMK Encrypted plugin, test cases file

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/efs/efsCmkEncrypted.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            if (!listAliases || listAliases.err || !listAliases.data) {
+                helpers.addResult(results, 3,
+                    `Unable to query for KMS aliases: ${helpers.addError(listAliases)}`,
+                    region);
+                return rcb();
+            }
+
+            var aliasId;
+            var kmsAliases = {};
+            var ckmUnencryptedEFS = [];
```
Comment (dipsubha06) [inline plugins/aws/efs/efsCmkEncrypted.js:57]: Minor comment. would it be cmkUnencryptedEFS ?

**Review by dipsubha06 — APPROVED** [review]

**Review by giorod3 — CHANGES_REQUESTED** [review]
just the one comment from subhadip

File: plugins/aws/efs/efsCmkEncrypted.js (hunk 1 above)
Comment (AkhtarAmir) [reply plugins/aws/efs/efsCmkEncrypted.js:57]: Yeah I believe this is a typo. I'll fix it.

Comment (AkhtarAmir) [reply plugins/aws/efs/efsCmkEncrypted.js]: Updated

**Pull request merged**